### ISSUE TITLE
Nested layouts in WebC

### ISSUE THREAD

**Issue by zachleat** [11ty/eleventy-plugin-webc#28]
Reported by https://fosstodon.org/@catgirl@catgirlin.space/109366609660374052

page.webc:
```
---
layout: layout.webc
---
```

layout.webc:
```
---
layout: base.webc
---
<template webc:nokeep @raw="content"></template>
```

base.webc:
```
<!doctype html>
<html lang="en">
	<head>
		<meta charset="utf-8">
		<meta name="viewport" content="width=device-width, initial-scale=1.0">
		<meta name="description" content="">
		<title></title>
		<style @raw="getCss(page.url)"></style>
	</head>
	<body>
		<template webc:nokeep @raw="content"></template>
	</body>
</html>
```

Verified that the `<style>` tag is missing!

**Comment by zachleat** [comment]
Fixed this one in core, it’ll ship with 2.0.0-canary.18!

Please retest after the next canary ships early next week!

**Comment by zachleat** [comment]
https://github.com/11ty/eleventy/issues/2654
